wdl-lint/src/util.rs

```diff
@@ -8,12 +8,14 @@ use wdl_ast::SyntaxKind;
 /// whitespace.
 pub fn is_inline_comment(token: &Comment) -> bool {
     if let Some(prior) = token.syntax().prev_sibling_or_token() {
-        return prior.kind() != SyntaxKind::Whitespace
-            || !prior
-                .as_token()
-                .expect("should be a token")
-                .text()
-                .contains('\n');
+        let not_whitespace = prior.kind() != SyntaxKind::Whitespace;
+        let contains_newline = prior
+            .as_token()
+            .expect("should be a token")
```

Comment:
Previously this `expect` was being short-circuited when the syntax kind was not whitespace; I fear we may now be panicking if the previous sibling is a node.

Reply:
Ah you're right. I wonder why that didn't cause any test/gauntlet issues?

Reply:
We probably don't have any tests that place a comment directly after a non-whitespace token (probably should add one). I doubt we'd encounter it in the wild either (plus we only lint a select few repos).